dist/index.js

```diff
@@ -8,6 +8,11 @@ module.exports = scan
 module.exports.cli = scanCLI
 module.exports.customScanner = scanUsingCustomSonarQubeScanner
 
+let foo
+if (foo == '') {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Either remove or fill this block of code. ([javascript:EmptyBlock](https://sonarcloud.io/organizations/default/rules#rule_key=javascript%3AEmptyBlock))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=sonarqube-scanner&pullRequest=38&issues=AWLNyHlbjr7wspRoiHiP&open=AWLNyHlbjr7wspRoiHiP)

[ISSUE_KEY]: AWLNyHlbjr7wspRoiHiP